assets/js/dashboard/stats/graph/top-stats.js

```diff
@@ -44,10 +44,14 @@ export default class TopStats extends React.Component {
     let statName = stat.name.toLowerCase()
     statName = stat.value === 1 ? statName.slice(0, -1) : statName
 
+    const { topStatData, lastLoadTimestamp } = this.props
+    const showingImported = topStatData && topStatData.imported_source && topStatData.with_imported
```

Comment:
Minor suggestion, feel free to ignore :)

```suggestion
    const showingImported = topStatData?.imported_source?.with_imported
```

Reply:
That wouldn't work, since `with_imported` is a property of `topStatData`, not `imported_source`, but I did use optional chaining here: https://github.com/plausible/analytics/pull/2738/commits/ab8601a2f31225c77ce179305ac09889c528294b